reference/sites.v3.yml

```diff
@@ -585,6 +596,36 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/CertificateResponse'
+    put:
+      summary: Upsert a Site's SSL/TSL Certificate Information
+      operationId: put-site-id-certificate
+      tags:
+        - Site Certificate
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/InstallCertificateData'
+        description: ''
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/EmptyResponse'
+            Example:
+              examples:
+                response:
+                  value: |-
+                    {
+                        "data": {},
+                        "meta": {}
+                    }
+      description: |-
+        - If `url` is not supplied, certificate would be saved for given site "primary" url
+        - `url` must be a valid URL within provided `site_id`
+        - Given endpoint works as UPSERT, supplying SSL certificate for a domain that already has a certificate connected would update SSL certificate for given domain'
```

Comment:
```suggestion
        - Use caution. Because this endpoint upserts, supplying an SSL certificate for a domain that already has a certificate connected overwrites the domain’s extant certificate.'
```